.travis.yml

```diff
@@ -3,11 +3,9 @@ osx_image: xcode8.3
 
 before_script:
   - bundle install
-  - cd Bootstrap ; bundle exec pod install --repo-update ; cd ..
+  - cd Bootstrap ; bundle exec pod install ; cd ..
```

Comment:
I'd live with it till something breaks, it would possibly happen when you update your RxSwift deps for example